server/controllers/message.js

```diff
@@ -0,0 +1,35 @@
+const Message = require("../models/Message");
+const Conversation = require("../models/Conversation");
+const asyncHandler = require("express-async-handler");
+
+// @route  GET /messages/:conversationId
+// @desc Get all messages for a particular conversation
+// @access Private
+exports.messagesForAConversation = asyncHandler(async (req, res) => {
+  const messages = await Message.find({
+    conversation: req.params.conversationId,
```

Comment:
Optional: You can perform path parameter validation and return a `400` if it is missing